cypress/support/api/servicePoints.js

```diff
@@ -48,15 +48,16 @@ Cypress.Commands.add('deleteServicePoint', (id) => {
   });
 });
 
-Cypress.Commands.add('addServicePointToUser', (servicePointId, userId) => {
+Cypress.Commands.add('addServicePointToUser', (servicePointIds, userId, defaultServicePointId) => {
```

Comment:
move to related fragment

Reply:
need to add TODO and Jira ticket to fix currently known issue with Cypress.env variable 'servicePoints'

Reply:
I created task in Jira FAT-2052

Reply:
Done